src/environment.js

```diff
@@ -33,9 +33,9 @@ export class Environment {
       case 'development': {
         this.flag = 'dev';
         // eslint-disable-next-line max-len
-        this.form.location = 'https://api.forms.platforms.qld.gov.au/dev-tzkqydhwrjrviss/labelbuster';
+        this.form.location = 'https://api.forms.platforms.qld.gov.au/lighthousedev-tzkqydhwrjrviss/labelbuster';
```

Comment:
can't have this for our dev branch.
why did not you add another switch case for your url so it can co-exist.

Reply:
line 12 could have a base includes lighthouse for the url and swap your form location. 
or
add a global variable check for an object to override form.location, base.baseLocation, form.adminEmail and flag so that dev work per developer can be split and not interfere with dev/uat/prod path.